packages/formation-react/src/components/AlertBox/AlertBox.jsx

```diff
@@ -63,7 +63,6 @@ class AlertBox extends React.Component {
         ref={ref => {
           this._ref = ref;
         }}
-        role="alert"
```

Comment:
Could setting `role` be done by passing in a property for the cases where an AlertBox needs a role?